### PR TITLE
Release 3.3.0

### PR DESCRIPTION
**Pull request by Goose-Of-War**
# Stuff covered

## Features
- ### Governor Portal
	- Poll Management Page
	- Poll Management Page
- ### Login Page Redirect
	- Redirect to the URL which prompted the login response
- ### JSON User
	- Can be used to test user functions when there are no users
- ### Newsletter Pagination
	- Newsletters have page navigators and buttons
	- Share specific page link 

## Enhancements
- ### Routers
	- Using `readdir` instead of writing a thesis long code
	- Error Page shows its own page title
- ### Submissions Page
	- Uses database instead of the Google Form
- ### Page Descriptions
	- Now, the embeds don't just display "The MASK Website"
- ### Query String
	- Used for IDs in gov-portal
	- Will be used more moving forward
- ### Icon Updates

## Bug Fixes
- ### Poll
	- Input Trimming
	- Percentage round-off bug
	- Margin
	- Mozilla Compatibility
- ### Lint
	- Templates: Quotes
	- Templates: Whitespaces
- ### Background
	- Background size remains fixed even when page size changes

## Documentation
- Migration to a separate folder
- README update (for new routers and more)

### CODE REVIEW

**Comment by nishkalprakash** [comment]
+ newsletter scrolling is fine
+ and pagination is working like it should.
+ Share button yeeted, and should be removed from this PRs description
+ Redirect is tested and is working
@gooselets plz check for gov portal bugs and json user bugs

**Comment by Goose-Of-War** [comment]
> * Share button yeeted, and should be removed from this PRs description
 
I mean, you technically can share the link of a specific page

File: templates/govportal/poll-management.njk
Comment (ItsAnkan) [inline templates/govportal/poll-management.njk:1]: Inconsistent spacing between the buttons
image

File: templates/govportal/post-management.njk
Comment (ItsAnkan) [inline templates/govportal/post-management.njk:1]: Same for this as Poll Management

**Review by SachdevJai — APPROVED** [review]
:pray:

**Pull request merged**